test/makeChange_test.js

```diff
@@ -0,0 +1,53 @@
+import { expect } from 'chai'
+import makeChange from '../src/makeChange'
+
+describe('makeChange()', function(){
+
+  it('should be a function', function(){
+    expect(makeChange).to.be.a('function')
+  })
+
+  it('returns an object with all coin types (quarters, dimes, nickels, and pennies)', function(){
+    const change = makeChange({price: 100, amountGiven: 100})
+    expect(change).to.be.an('object')
+    expect(change).to.have.keys('quarters', 'dimes', 'nickels', 'pennies')
+    expect(change).to.deep.equal({
+      quarters: 0,
+      dimes: 0,
+      nickels: 0,
+      pennies: 0,
+    })
+  })
+
+  it('returns correct change', function(){
+    expect(makeChange({price: 100, amountGiven: 141})).to.deep.equal({
+      quarters: 1,
+      dimes: 1,
+      nickels: 1,
+      pennies: 1,
+    })
+  })
+
+  it('minimizes the number of coins given by using the most high-value coins', function(){
```

Comment:
This file contains a good variety of cases. And all of your test descriptions that I've seen thus far are very descriptive and helpful.